client/src/ui/track/ApplicationDeleteModal.tsx

```diff
@@ -56,14 +56,58 @@ export default function ApplicationDelete({
           gap: "0px 30px",
         }}
       >
-        <Typography id="simple-modal-title">
+        <Typography id="simple-modal-title" variant="h6">
           Are you sure you want to delete this application?
         </Typography>
         <Typography id="simple-modal-description">
           This action cannot be undone.
         </Typography>
-        <Button onClick={handleClose}>Cancel</Button>
-        <Button onClick={handleDelete}>Delete</Button> 
+        <Box
```

Comment:
Can we add padding to the modal so the content is not as close to the border. It feels claustrophobic with the lack of spacing.

Reply:
Fixed!